lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -247,15 +247,15 @@ public function getClobTypeDeclarationSQL(array $field)
         if (! empty($field['length']) && is_numeric($field['length'])) {
             $length = $field['length'];
 
-            if ($length <= static::LENGTH_LIMIT_TINYTEXT) {
```

Comment:
Also here: BC compliant? Are there overrides in subclasses?

Reply:
Nope, non of them overridden in any subclass